src/app/workbench/sidebar/CreateNextWorkbenchSidebar.tsx

```diff
@@ -116,6 +128,7 @@ export function CreateNextWorkbenchSidebar({ workbench }: ICreateNextWorkbenchSi
                         columnDescs: [],
                         data: {},
                         entityId: relationList[0].targetEntity,
+                        formatting: { idColumn: 'id' },
```

Comment:
Can we always ensure that there is a column with the name `id`?

Reply:
woops, did not mean to commit this.

Reply:
That's why we do code reviews. 😉

Reply:
thanks for the catch :-)